[2023-09-19 07:38:11,708] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T00:37:00+00:00 [queued]>
[2023-09-19 07:38:11,968] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T00:37:00+00:00 [queued]>
[2023-09-19 07:38:11,968] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:38:11,968] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2023-09-19 07:38:11,969] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:38:12,023] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2023-09-19T00:37:00+00:00
[2023-09-19 07:38:12,033] {standard_task_runner.py:52} INFO - Started process 463 to run task
[2023-09-19 07:38:12,090] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', '2023-09-19T00:37:00+00:00', '--job-id', '40', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_pipeline.py', '--cfg-path', '/tmp/tmptxr2epf7', '--error-file', '/tmp/tmpv_7vqc51']
[2023-09-19 07:38:12,172] {standard_task_runner.py:77} INFO - Job 40: Subtask fetch_weather_data
[2023-09-19 07:38:12,807] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T00:37:00+00:00 [running]> on host 92beb7708765
[2023-09-19 07:38:13,335] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data_pipeline
AIRFLOW_CTX_TASK_ID=fetch_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-19T00:37:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-19T00:37:00+00:00
[2023-09-19 07:38:15,277] {python.py:151} INFO - Done. Returned value was: True
[2023-09-19 07:38:15,816] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, execution_date=20230919T003700, start_date=20230919T073811, end_date=20230919T073815
[2023-09-19 07:38:16,271] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-19 07:38:16,548] {local_task_job.py:151} INFO - Task exited with return code 0
